docs/installation.md

```diff
@@ -71,6 +71,18 @@ sudo PIPX_HOME=/opt/pipx PIPX_BIN_DIR=/usr/local/bin pipx install PACKAGE
 # Example: $ sudo PIPX_HOME=/opt/pipx PIPX_BIN_DIR=/usr/local/bin pipx install cowsay
 ```
 
+!!! note
+
+    After version 1.2.0, the default pipx paths have been moved from `~/.local/pipx` to `$XDG_DATA_HOME/pipx`, which is typically `~/.local/share/pipx`, to ensure compatibility with the [XDG base directory specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html#variables).
```

Comment:
How does Windows apply here?

Reply:
I can't test on Windows. Where does pipx store its files on Windows?

Reply:
Worthwhile to ask `platformdirs`. Or maybe they have this in the documentation already.

Reply:
Come to think of it Macs are also an issue. Maybe instead of explicitly mentioning XDG this part should just redirect users to platformdirs documentation as the canonical source.

Reply:
> Come to think of it Macs are also an issue. Maybe instead of explicitly mentioning XDG this part should just redirect users to platformdirs documentation as the canonical source.

I don't have a Mac and I'm not familiar with it.

I can only test this PR on Linux, I'd be happy if someone could help and test the PR on other platforms. 

Perhaps it's best to limit this change to Linux for now, if it can't be tested.

Reply:
The code is fine, only the documentation is problematic.

Reply:
Sorry for the late reply. Can you tell me the default pipx on Windows and Mac **before** this change, so I can update this documentation accordingly?

Reply:
Before the change they are exactly the same as on Linux!

Reply:
Understood, then I think the current change to the documentation is fine.